core/src/table/table.module.css

```
@@ -51,3 +51,53 @@ in native HTML */
 .container tr:last-child td {
 	border-bottom-style: none;
 }
+
+/* Cell spacing */
+
+/* Medium */
+.medium th,
+.medium td {
+	padding: 12px 8px;
+}
+
+.medium td:first-child,
+.medium th:first-child {
+	padding-left: 16px;
+}
+
+.medium td:last-child,
+.medium th:last-child {
+	padding-right: 16px;
+}
+
+.small th,
+.small td {
+	padding: 9px 6px;
```

We prefer 8-based sizing

```suggestion
	padding: 8px 6px;
```

---

I think we should use global variables in here;